CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -14,358 +14,361 @@ static function array<X2DataTemplate> CreateTemplates()
-	CreateRecoverDarkEvent(Templates);
-	CreateCounterDarkEvent(Templates);
+	CreateDarkEventWaitActivity(Templates);
+	CreateGenericWaitActivity(Templates);
 	
-	CreatePrepareFactionJB(Templates);
-	CreateJailbreakSoldier(Templates);
-	CreateJailbreakChosenSoldier(Templates);
-	CreateJailbreakFactionSoldier(Templates);
+	CreateSupplyConvoy(Templates);
+	CreateSupplyExtract(Templates);
+	CreateSecureUFO(Templates);
+	CreateCaptureDVIP(Templates);
 	
-	CreateRecoverUFO(Templates);
+	CreatePreparePersonnel(Templates);
+	CreatePrepareFactionJB(Templates);
 	CreatePrepareUFO(Templates);
-	CreateLandedUFO(Templates);
-	
-	CreateCommanderSupply(Templates);
-	CreateSupplyRaid(Templates);
-	
 	CreatePrepareFacility(Templates);
-	CreateFacilityInformant(Templates);
-	
-	CreateGatherIntel(Templates);
-	CreateGatherSupplies(Templates);
-	
-	CreateIntelRescue(Templates);
-	CreateSupplyRescue(Templates);
-	
+
 	return Templates;
 }
 
-static function CreateRecoverPersonnel (out array<X2DataTemplate> Templates)
+
+
+/////////////////////////
+/// Random Activities ///
+/////////////////////////
+
+static function CreateIntelligenceGeneric(out array<X2DataTemplate> Templates)
 {
-	local X2ActivityTemplate_Infiltration Activity;
+	local X2ActivityTemplate_Assault ActivityAssault;
+	local X2ActivityTemplate_Infiltration ActivityInfil;
 	local X2CovertActionTemplate CovertAction;
 	
-	CovertAction = class'X2StrategyElement_InfiltrationActions'.static.CreateInfiltrationTemplate('CovertAction_RecoverPersonnelInfil', true);
-	Activity = CreateStandardInfilActivity(CovertAction, "RecoverPersonnel", "ResOps", "img:///UILibrary_XPACK_Common.MissionIcon_ResOps");
-
-	Activity.bNeedsPOI = true;
+	`CREATE_X2TEMPLATE(class'X2ActivityTemplate_Assault', ActivityAssault, 'Activity_IntelligenceAssault');
 	
-	Activity.MissionRewards.AddItem('Reward_Rumor');
-	Activity.MissionRewards.AddItem('Reward_SmallIntel');
-	Activity.GetMissionDifficulty = GetMissionDifficultyFromMonth;
-	Activity.WasMissionSuccessful = class'X2StrategyElement_DefaultMissionSources'.static.OneStrategyObjectiveCompleted;
-	Activity.AvailableSound = "GeoscapeFanfares_GuerillaOps";
+	ActivityAssault.OverworldMeshPath = "UI_3D.Overwold_Final.RadioTower";
+	ActivityAssault.UIButtonIcon = "img:///UILibrary_StrategyImages.X2StrategyMap.MissionIcon_ResHQ";
+	ActivityAssault.MissionImage = "img:///UILibrary_XPACK_StrategyImages.CovertOp_Recover_X_Intel";
+	ActivityAssault.Difficulty = 1;
 	
-	Templates.AddItem(CovertAction);
-	Templates.AddItem(Activity);
-}
-
-static function CreatePreparePersonnel (out array<X2DataTemplate> Templates)
-{
-	local X2ActivityTemplate_CovertAction Activity;
-	local X2CovertActionTemplate CovertAction;
+	ActivityAssault.ActivityTag = 'Tag_Intelligence';
+	ActivityAssault.bNeedsPOI = true;
+	ActivityAssault.MissionRewards.AddItem('Reward_Rumor');
+	ActivityAssault.MissionRewards.AddItem('Reward_SmallIntel');
+	ActivityAssault.GetMissionDifficulty = GetMissionDifficultyFromMonth;
```

Comment:
Why not `GetMissionDifficultyFromMonthPlusTemplate`? I thought we decided that assaults should be harder.

(This applies to other assaults below)